index.html

```
@@ -3107,7 +3107,7 @@ <h2>Keyboard Shortcuts</h2>
     </tr>
 
     <tr>
-      <td class="py-2">C</td>
+      <td class="py-2">C</td
```

HTMLHint (1.9.2)

[error] Tag must be paired, missing: [ </td> ], start tag match failed [ <td class="py-2"> ] on line 3110.

(tag-pair)